packages/core/src/model/chain/redditArbitrum.ts

```diff
@@ -0,0 +1,21 @@
+import { Chain } from '../../constants'
+import { getAddressLink, getTransactionLink } from '../../helpers/chainExplorerLink'
+
+const testnetRedditscanUrl = 'https://testnet.redditspace.com'
+
+export const RedditTestnet: Chain = {
+  chainId: 5391184,
+  chainName: 'RedditTestnet',
+  isTestChain: true,
+  isLocalChain: false,
+  multicallAddress: '0xFf79F208541DdF9dA6632dD851a269BbfA97d9aD',
+  multicall2Address: '',
+  rpcUrl: 'https://testnet.redditspace.com/rpc',
+  blockExplorerUrl: testnetRedditscanUrl,
+  getExplorerAddressLink: getAddressLink(testnetRedditscanUrl),
```

Comment:
I think that something is broken with that, i pasted explorer address link with multicall address and there i had transactions with that address.
Either address link is wrong or multicall address, please check it.

Reply:
I have fixed it, multicall: `0x722db82dea58c880d03b87885053f206f1b37136`